pkg/apis/v1beta1/aksnodeclass.go

```diff
@@ -24,8 +24,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type FIPSMode string
```

Comment:
I made this mistake too in another PR of mine:

We have 2 AKSNodeClass copies, one v1alpha2, one v1beta1. I'm wondering if we should be putting this in both?

@tallaxes may have a plan for when we remove the v1alpha2 version (have we made any sort of announcement or release note on when that's going to happen?)